stats/DataAggregation.md

```diff
@@ -0,0 +1,67 @@
+# Data Aggregation API Overview
+The stats library allows users to define measurements how data are aggregated and broken down.
+The core data types used are:
```

Comment:
I still think we should leave Aggregation Window as a concept; even if we should leave it out of the public API for simplicity, z-page and LB support make it likely that multiple implementations will want to keep the concept internally, and if so it is still useful to standardize terminology.

Reply:
We will add a different doc explaining implementation high-level design which will address the window issue.